### PR TITLE
Changes made to fix the issue caused by wrong field name.

### PR DESCRIPTION
**Pull request by mazmat-panw**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
In XSOAR 6.6, when REST API is used to fetch the details of all the playbooks the field 'id' does not contain the name of the playbook anymore as it did in previous versions. So the code had to be modified and now I am using the filed called 'nameRaw' to get the name of the playbook.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ X] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CreatePlbkDoc/ReleaseNotes/1_1_0.md [hunk 1]
```diff
@@ -1,5 +1,4 @@
 
 #### Scripts
 ##### CreatePlbkDoc
-- Fixed an issue when getting a list of playbooks with given name, to get the exact playbook.
-- Improved implementation of using dictionaries, change square brackets to get method.
+- Fixed the issue that was produced due ot change in the XSOAR ver 6.6. Using the field 'nameRaw' instead of 'id'.
```
Comment (yaakovi) [inline Packs/CreatePlbkDoc/ReleaseNotes/1_1_0.md:4]: revert this change.

File: Packs/CreatePlbkDoc/.pack-ignore [hunk 2]
```diff
@@ -2,6 +2,7 @@
 ignore=RM106
 
 [file:pack_metadata.json]
-ignore=PA129
-
+ignore=PA129, PA126
```
Comment (yaakovi) [inline Packs/CreatePlbkDoc/.pack-ignore:5]: `PA126` is not ignorable (This is just a warning anyway).
```suggestion
ignore=PA129
```

File: Packs/CreatePlbkDoc/.pack-ignore [hunk 3]
```diff
@@ -2,6 +2,7 @@
...
 
+[file:CreatePlbkDoc.yml]
+ignore=DO106
```
Comment (yaakovi) [inline Packs/CreatePlbkDoc/.pack-ignore:8]: Need to update the docker image in the script yml file.
This error cannot be ignored.
```suggestion
```

File: Packs/CreatePlbkDoc/Scripts/CreatePlbkDoc/CreatePlbkDoc.py [hunk 4]
```diff
@@ -264,7 +264,7 @@ def main():
 
     while j < len(retVal):
 
-        PlaybookNameTempStr = retVal[j].get('id')
+        PlaybookNameTempStr = retVal[j].get('nameRaw')
```
Comment (yaakovi) [inline Packs/CreatePlbkDoc/Scripts/CreatePlbkDoc/CreatePlbkDoc.py:267]: check what is the current XSOAR version (use the `is_demisto_version_ge` function):
if the version is below 6.6, use `id`.
otherwise, use `nameRaw`.

**Review by yaakovi — CHANGES_REQUESTED** [review]

**Comment by yaakovi** [comment]
> check what is the current XSOAR version (use the `is_demisto_version_ge` function): if the version is below 6.6, use `id`. otherwise, use `nameRaw`.

@mazmat-panw do you need help with this one?

**Comment by mazmat-panw** [comment]
Hi Shai,
As I mentioned earlier in my email, we do not need to make that change as
the field 'nameRaw' exists with the correct value in xsoar 6.2, 6.5 and
6.6. As long as I am using nameRaw i should be good for the earlier
versions as well.
Please let me know if there is anything needed from me before you could
merge.

Thanks.


Mahmood Azmat  | Solutions Architect

Palo Alto Networks  |  3901 N Dallas Pkwy  |  Plano, TX 75093  |  USA

Ph: 214-380-5945  | Support: +1.866.898.9087

Email: ***@***.***  | www.paloaltonetworks.com

**Review by yaakovi — APPROVED** [review]
